src/officehours_api/notifications.py

```diff
@@ -12,10 +12,18 @@
 
 from officehours_api.models import Queue, Meeting, MeetingStatus
 
-logger = logging.getLogger(__name__)
-
-twilio = TwilioClient(settings.TWILIO_ACCOUNT_SID, settings.TWILIO_AUTH_TOKEN)
-
+def initialize_twilio():
+    logger = logging.getLogger(__name__)
```

Comment:
line 16 shall be put outside of the initialize_twilio() function. 
logger does not need to be a return object

Reply:
Pulled it out of the function